google/cloud/spanner_v1/_pandas_helpers.py

```diff
@@ -0,0 +1,76 @@
+import logging
+import six
+
+class pandas_df():
+    
+    def pd_dataframe(self):
+        """Returns the response in a pandas dataframe of the StreamedResultSet object"""
+        try :
+            from pandas import DataFrame
+        except ImportError:
+            logging.error("Pandas module not found. It is needed for converting query result to Dataframe.\n Try running 'pip3 install pandas'")
```

Comment:
The readable error message is nice, but this still needs to raise an exception. Ideally, it would re-raise the exception so that the stack trace is preserved. https://stackoverflow.com/a/792163/101923

Reply:
Modified